### PR TITLE
typo in entrypoint link name

### PR DESCRIPTION
**Pull request by bscheshirwork**
so... entrypoint don't exec (full path needed)?

### CODE REVIEW

**Comment by bscheshirwork** [comment]
comment please

**Comment by yosifkit** [comment]
I am not sure what this is fixing.  `/usr/local/bin/` is in the `PATH` so `docker-entrypoint.sh` should be found correctly by docker when the container is started.

**Comment by bscheshirwork** [comment]
I think real path is clear... In static container.

Full path needed

in some cases 

i.e. exec from scripts...

File: 8.0/Dockerfile [hunk 1]
```diff
@@ -68,8 +68,8 @@ RUN sed -Ei 's/^(bind-address|log)/#&/' /etc/mysql/mysql.conf.d/mysqld.cnf \
 VOLUME /var/lib/mysql
 
 COPY docker-entrypoint.sh /usr/local/bin/
-RUN ln -s usr/local/bin/docker-entrypoint.sh /entrypoint.sh # backwards compat
-ENTRYPOINT ["docker-entrypoint.sh"]
+RUN ln -s /usr/local/bin/docker-entrypoint.sh /entrypoint.sh # backwards compat
+ENTRYPOINT ["/entrypoint.sh"]
```
Comment (tianon) [inline 8.0/Dockerfile]: A relative symlink is relative to where the file resides, so keeping this root-level compatibility symlink relative makes sure that even if the rootfs is mounted elsewhere, this symlink still points to the right thing.

```console
$ docker pull mysql:8.0
8.0: Pulling from library/mysql
Digest: sha256:e95c17f555f6db37e56b37e47b80ce089a344ef4b32d284ca196a37a38c298f2
Status: Image is up to date for mysql:8.0
$ docker run -it --rm mysql:8.0 bash
root@fa8e667b8ad5:/# ls -l /
total 68
drwxr-xr-x   2 root root 4096 Apr 11 22:40 bin
drwxr-xr-x   2 root root 4096 Dec 28 17:42 boot
drwxr-xr-x   5 root root  360 Apr 24 16:53 dev
drwxr-xr-x   2 root root 4096 Mar 21 20:57 docker-entrypoint-initdb.d
lrwxrwxrwx   1 root root   34 Apr 11 22:41 entrypoint.sh -> usr/local/bin/docker-entrypoint.sh
drwxr-xr-x  58 root root 4096 Apr 24 16:53 etc
drwxr-xr-x   2 root root 4096 Dec 28 17:42 home
drwxr-xr-x  11 root root 4096 Apr 11 22:40 lib
drwxr-xr-x   2 root root 4096 Mar 20 23:27 lib64
drwxr-xr-x   2 root root 4096 Mar 20 23:26 media
drwxr-xr-x   2 root root 4096 Mar 20 23:26 mnt
drwxr-xr-x   2 root root 4096 Mar 20 23:26 opt
dr-xr-xr-x 494 root root    0 Apr 24 16:53 proc
drwx------   2 root root 4096 Mar 20 23:26 root
drwxr-xr-x   4 root root 4096 Apr 11 22:41 run
drwxr-xr-x   2 root root 4096 Mar 20 23:29 sbin
drwxr-xr-x   2 root root 4096 Mar 20 23:26 srv
dr-xr-xr-x  12 root root    0 Mar 24 17:03 sys
drwxrwxrwt   2 root root 4096 Apr 11 22:41 tmp
drwxr-xr-x  23 root root 4096 Apr 11 22:41 usr
drwxr-xr-x  21 root root 4096 Apr 11 22:40 var
root@fa8e667b8ad5:/# readlink -f /entrypoint.sh 
/usr/local/bin/docker-entrypoint.sh
root@fa8e667b8ad5:/# which docker-entrypoint.sh
/usr/local/bin/docker-entrypoint.sh
root@fa8e667b8ad5:/# 
```

Comment (bscheshirwork) [inline 8.0/Dockerfile]: "if the rootfs is mounted elsewhere"
`usr/local/bin/docker-entrypoint.sh`
still correct?

Comment (tianon) [inline 8.0/Dockerfile]: Yes, here's another example showing that:
```console
~/temp/mysql $ docker create --name temp mysql:8.0
e11a8d26ead5c8517083e41767ecc72126a3ed1913ba9d234960b867d10ddc7f
~/temp/mysql $ docker export temp | tar -x
~/temp/mysql $ pwd -P
/home/tianon/temp/mysql
~/temp/mysql $ ls -l entrypoint.sh
lrwxrwxrwx 1 tianon tianon 34 Apr 11 15:41 entrypoint.sh -> usr/local/bin/docker-entrypoint.sh
~/temp/mysql $ readlink -f entrypoint.sh
/home/tianon/temp/mysql/usr/local/bin/docker-entrypoint.sh
~/temp/mysql $ 
```

**Pull request closed without merging**